src/tooltip/src/Tooltip.js

```diff
@@ -183,7 +183,10 @@ Tooltip.propTypes = {
   showDelay: PropTypes.number,
 
   /**
-   * When True, manually show the Tooltip.
+   * Controls whether the Tooltip is shown or not.
+   * - When `true`, the component is manually shown.
```

Comment:
Same here (idk if I can point to all these with all this a single comment, sorry)